### PR TITLE
Reqs parsed from docstrings

### PR DESCRIPTION
**Pull request by matuu**

### CODE REVIEW

**Comment by gilgamezh** [comment]
:+1: 
We have to solve the merge conflicts and wait for @facundobatista +1

File: fades/parsing.py [hunk 1]
```diff
@@ -106,6 +107,32 @@ def _parse_content(fh):
...
 
 
+def _parse_docstring(fh):
+    """Parse the docstrings of a script to find marked dependencies."""
+    requirements = []
+    # get docstring into source file
+    ds_parsed = ast.parse(fh.read())
+    ds_texts = [ds_parsed]
+    ds_texts.extend([node for node in ds_parsed.body if isinstance(node, ast.FunctionDef) or
+                     isinstance(node, ast.ClassDef) or isinstance(node, ast.Module)])
```
Comment (facundobatista) [inline fades/parsing.py]: Do we want to get the docstrings for everything? Or just the module? I vote for "module only"

**Comment by facundobatista** [comment]
Gilgamezh, Matías, I added an inline comment about somethin we need to discuss... the rest, I like it :)

File: fades/parsing.py (hunk 1 above)
Comment (gilgamezh) [inline fades/parsing.py]: :+1: module only!

**Comment by matuu** [comment]
I fixed this and I opened (#111) another pull request.

**Pull request closed without merging**